install_config/aggregate_logging.adoc

```diff
@@ -1006,6 +1006,11 @@ $ oc patch dc/logging-es-<suffix> \
 . Once these steps are taken, a local host mount can be applied to each replica as in this example
 (where we assume storage is mounted at the same path on each node) (change to `--selector component=es-ops` for Ops cluster):
 +
+=======
+Once you have completed these steps, you can apply a local host mount to each
+replica. The following example assumes storage is mounted at the same path
+on each node.
+
```

Comment:
@jhoyt-rh - missing ending ======.

This is not rendering correctly: https://docs.openshift.com/container-platform/3.11/install_config/aggregate_logging.html